### PR TITLE
refactor(frontend): split the backend from the frontend folder to refactor

### LINKED ISSUE #13: [ORG]: refactor the file structure
### Description

Refactor the file structure to differentiate frontend and backend file

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by YuanhuanO**
Closes #13 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed
- [x] Branch name follows `type/descript` (e.g. `feature/add-llm-agents`)
- [x] Ready for code review

## ℹ Additional Information

1. Seperate frontend and backend by using frontend and gpu_bartender/sever in the root dir
2. Change the programming language for backend from typescript into python
3. Refactor and reconstruct the Webpack configuration
4. Update package.json setting

### CODE REVIEW

**Review by lwaekfjlk — CHANGES_REQUESTED** [review]
Could you pull with main and solve the conflicts? Thanks

File: gpu_bartender/.DS_Store
Comment (lwaekfjlk) [inline gpu_bartender/.DS_Store:1]: this file should not be inlcuded.

**Comment by lwaekfjlk** [comment]
@YuanhuanO I checked other repo and it shows that it is not common to upload node_modules to the github repo?

**Comment by lwaekfjlk** [comment]
check this line: https://github.com/OpenDevin/OpenDevin/blob/main/.gitignore#L176

**Comment by YuanhuanO** [comment]
I

> it shows that it is not common to upload node_modules to the github rep

Thanks for mentioning that!

**Pull request merged**